.github/workflows/issues.yml

```diff
@@ -3,13 +3,14 @@
   
   on:
     issues:
-      types: [opened]
+      types: [opened, reopened, closed]
   permissions: 
     issues: write
     contents: read
   jobs:
     jira_task:
       name: Create Jira issue
+      if: github.event_name == 'issues' && github.event.action == 'opened'
```

Comment:
knitpit: do we need to check github.event_name == 'issues' ? 
it looks like it's the only "on" trigger

Reply:
no good catch. I will remove it

Reply:
updated